mkdocs.yml

```diff
@@ -55,6 +55,8 @@ nav:
       - configuration/integrations/bitbucket-server.md
       - configuration/integrations/email.md
       - configuration/integrations/troubleshoot.md
+    - "Secrets":
+      - configuration/secrets/troubleshoot.md
```

Comment:
I will like to place this new troubleshooting information in a better place (just so we don't have several scattered "troubleshooting" pages). But I'll wait until we settle all the troubleshooting info that we will provide to propose a new / simpler structure.

Reply:
I opened the pull request https://github.com/codacy/chart/pull/391 to do this refactoring.